src/wexond/renderer/app/components/App/index.tsx

```diff
@@ -3,26 +3,27 @@ import * as React from 'react';
 import { createGlobalStyle, ThemeProvider } from 'styled-components';
 
 import { Style } from '~/renderer/app/style';
-import { Toolbar } from '../Toolbar';
+import Toolbar from '../Toolbar';
 import { ipcRenderer } from 'electron';
 import { Line, StyledApp } from './style';
 import { WindowsButtons } from '../WindowsButtons';
 import { platform } from 'os';
 import { Overlay } from '../Overlay';
 import store from '../../store';
+import { App } from '../../models/app'
 
 const GlobalStyle = createGlobalStyle`${Style}`;
 
 window.onbeforeunload = () => {
   ipcRenderer.send('browserview-clear');
 };
 
-export const App = observer(() => {
+export default observer((data: App) => {
```

Comment:
Why this change ? Just to not have the same name as `App` in `models/app` ?

Reply:
Actually I just used the same logic as what was done in other Wexond components without even trying to think of something better...